packages/m2m-gateway/src/services/purposeService.ts

```diff
@@ -182,5 +235,129 @@ export function purposeServiceBuilder(clients: PagoPAInteropBeClients) {
 
       return toM2mGatewayApiPurposeVersion(createdVersion);
     },
+    async activatePurpose(
```

Comment:
Operation Id in the SRS is `activateDraftPurpose`. Let's change the service name as such